src/contracts/libraries/BridgingHelper.sol

```diff
@@ -15,4 +17,83 @@ library BridgingHelper {
     Vote_Results,
     Payload_Execution
   }
+
+  function decodeMessageWithType(
+    bytes memory messageWithType
+  ) internal pure returns (BridgingHelper.MessageType, bytes memory) {
+    return abi.decode(messageWithType, (BridgingHelper.MessageType, bytes));
+  }
+
+  function decodeStartProposalVoteMessage(
+    bytes memory message
+  ) internal pure returns (uint256, bytes32, uint24) {
+    return abi.decode(message, (uint256, bytes32, uint24));
+  }
+
+  function decodePayloadExecutionMessage(
+    bytes memory message
+  )
+    internal
+    pure
+    returns (uint40, PayloadsControllerUtils.AccessControl, uint40)
+  {
+    return
+      abi.decode(
+        message,
+        (uint40, PayloadsControllerUtils.AccessControl, uint40)
+      );
+  }
+
+  function decodeVoteResultMessage(
+    bytes memory message
+  ) internal pure returns (uint256, uint128, uint128) {
+    return abi.decode(message, (uint256, uint128, uint128));
+  }
+
+  function encodePayloadMessageForExecution(
```

Comment:
better to name `encodePayloadExecutionMessage`, as you did in corresponding decode